lambdas/services/dynamo_service.py

```diff
@@ -58,6 +58,20 @@ def query_service(
             logger.error(e)
             raise e
 
+    def simple_query(self, table_name: str, key_condition_expression):

```

Comment:
We should maybe think about the naming of some of these functions, to me they're not very clear. What is a simple query? the `key_condition_expression` is missing a type so it may make it hard for someone to know how to implement it